lib/indexer.rb

```diff
@@ -53,8 +53,8 @@ def merge_indexes
     cat(cdxj_files, tmp_path)
 
     # sort the tmp file into place
-    cmd = ['sort', tmp_path.to_s, '--output', index_path.to_s]
-    puts('Error while sorting') unless system(*cmd)
+    cmd = "LC_ALL=C sort #{tmp_path} --output \"#{index_path}\""
```

Comment:
Can you drop a comment in here about why this is necessary?

Reply:
Were you thinking here in GitHub or in the code?

Reply:
I went with a comment in the source code. I also added TMPDIR which really ought to be set as well, which I noticed when looking at was_robots_suite.